Formula/victoriametrics.rb

```diff
@@ -0,0 +1,76 @@
+class Victoriametrics < Formula
+  desc "Cost-effective and scalable monitoring solution and time series database"
+  homepage "https://victoriametrics.com/"
+  url "https://github.com/VictoriaMetrics/VictoriaMetrics/archive/v1.88.1.tar.gz"
+  sha256 "92803ae61f927b7173e25d4537c429bf3054edfa2f719bafc8fb9fcd411cab87"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV.deparallelize
+    mkdir_p buildpath/"src/github.com/VictoriaMetrics"
+    ln_sf buildpath, buildpath/"src/github.com/VictoriaMetrics/VictoriaMetrics"
```

Comment:
This shouldn't be needed with modern Go software

Reply:
Removed this lines.